sources/assets/bloodhound-ce/bloodhound-ce

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "##### EXEGOL SAYS ##### Listening on http://$(jq -r .bind_addr < /opt/tools/BloodHound-CE/bloodhound.config.json)"
+echo "#####             ##### Don't forget to run neo4j-start"
+echo "#####             ##### Login : admin"
+echo "#####             ##### Password : intial password will only be printed on the first run of bloodhound, it then needs to be changed in the Web UI"
+echo "#####             #####            if you lost the password, run bloodhound-ce-reset."
+
+if ! pg_isready -q; then
```

Comment:
Isn't there any equivalent for neo4j while we are at it ?

Reply:
We didn't implement this for bloodhound < bloodhound-ce, I wanted to keep the same behavior.
However, when the service and ports manager is implemented, I want to have neo4j managed when starting any version of bloodhound.
TL;DR: we can keep it that way for now, so that all versions of bloodhound behave the same. Will change later on with the srv manager utility being developped